opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -291,6 +291,15 @@ message Span {
     // dropped_attributes_count is the number of dropped attributes. If the value is 0,
     // then no attributes were dropped.
     uint32 dropped_attributes_count = 5;
+
+    // Flags, a bit field. 8 least significant bits are the trace flags as
+    // defined in W3C Trace Context specification. 24 most significant bits are reserved
+    // and must be set to 0. Readers must not assume that 24 most significant bits
+    // will be zero and must correctly mask the bits when reading 8-bit trace flag (use
+    // flags & TRACE_FLAGS_MASK).
+    // This value is required, as not setting it will be interpeted as 0 which is 
+    // a valid representation of all flags being present and set to 0.
+    fixed32 flags = 6;
```

Comment:
Can you move this closer to the ids?

Reply:
No problem, but it looks like the messages are currently sorted by "field number", and moving it up will break this pattern. Are we OK with that?